src/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/Base64Transforms.cs

```diff
@@ -263,4 +366,47 @@ protected virtual void Dispose(bool disposing)
             Dispose(false);
         }
     }
+
+    internal class ThrowHelper
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Really?

Reply:
Yes, it's intentional.

For "small" methods like `ToBase64Transform.TransformBlock` it's a 10...15% perf-win.
For larger methods like `FromBase64Transform.TransformFinalBlock` it's just a 1...2% perf-win.

As `TransformBlock` will be called more often than `TransformFinalBlock` I would stick to forcing the inline here.